김진/231206/자릿수-더하기.js

```diff
@@ -0,0 +1,7 @@
+function solution(n, sum = 0) {
+  const Arr = String(n).split("").map(Number);
```

Comment:
```suggestion
  const Arr = String(n).split("").map(Number);
```
.map(Number);이 부분에서 받아오는 벨류를 바로 호출하면 생략이 가능한걸 알고 계시다니 굳이네요